app.py

```diff
@@ -194,15 +198,29 @@ def user_signUpSuccess():
     else:
         return render_template("user_signUpSuccess.html")
 
-### /user_newTeacherPage (POST -> templateRendered)
-### user_newTeacherPage route, for the new teachers that need to select the number of assignments
-### POST: let teachers select number of assignments limited to their code limit
-@app.route("/user_newTeacherPage", methods = ["POST"])
-#@login_required
+
+# Helper Function
+# USAGE: Will assign a country to a teacher or return a message error
+def assign_helper(looking_for, type_of_committee, is_important, is_advanced, teacher):
+    if looking_for != 0:
+        available = modHelpers.stillAvailable(typeOfCom=type_of_committee, important=is_important, advanced=is_advanced)
+        if available >= looking_for:
```

Comment:
Would be nice to still assign the assignments available and then return an error message with the difference.

Reply:
It has been added!